arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1660 @@
+        "description": "Add an EventHub Consumer Group to an IotHub.",
+        "operationId": "IotHubResource_CreateEventHubConsumerGroup",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "eventHubEndpointName",
+          "in": "path",
+          "description": "The name of the event hub endpoint.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "name",
+          "in": "path",
+          "description": "The name of the consumer group.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "description": "The name of the resource group.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EventHubConsumerGroupInfo"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "delete": {
+        "tags": ["DELETE"],
+        "summary": "Delete EventHub Consumer Group for an IotHub.",
+        "description": "Delete EventHub Consumer Group for an IotHub.",
+        "operationId": "IotHubResource_DeleteEventHubConsumerGroup",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "eventHubEndpointName",
+          "in": "path",
+          "description": "The name of the event hub endpoint.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "name",
+          "in": "path",
+          "description": "The name of the consumer group.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "description": "The name of the resource group.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Object"
```

Comment:
Is the service returning an empty object or null? If it is null then do not describe the schema. just the description should be fine. is 204 an expected response?